tests/regression/24-octagon/03-previously_problematic_a.c

```diff
@@ -1,27 +0,0 @@
-// PARAM: --set solver td3 --enable ana.int.interval --enable exp.partition-arrays.enabled  --set ana.activated "['base','threadid','threadflag','expRelation','octagon','mallocWrapper']" --set exp.privatization none
-// These examples were cases were we saw issues of not reaching a fixpoint during development of the octagon domain. Since those issues might
-// resurface, these tests without asserts are included
```

Comment:
Could these also not be just converted to apron analysis tests? Keeping them around wouldn't hurt in these cases where the relational analysis is invoked in a bunch of different ways.

Reply:
Yes, I re-added the tests, changed them to use apron and modified the workflow to run them.